packages/react/src/TooltipV2/Tooltip.tsx

```diff
@@ -278,11 +278,17 @@ export const Tooltip = React.forwardRef(
       // Has trigger element or any of its children interactive elements?
       const isTriggerInteractive = isInteractive(triggerRef.current)
       const triggerChildren = triggerRef.current.childNodes
-      const hasInteractiveChild = Array.from(triggerChildren).some(child => {
-        return child instanceof HTMLElement && isInteractive(child)
+      // two levels deep
+      const hasInteractiveDescendant = Array.from(triggerChildren).some(child => {
```

Comment:
Could we use `hasInteractiveNodes` for this is do we need to have the hierarchy (e.g. one or two levels deep)? https://github.com/primer/react/blob/d25d58a7e8393c7e5e8e8f7fae09a7244777221f/packages/react/src/internal/utils/hasInteractiveNodes.ts#L34